creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -602,9 +610,34 @@ class SnapbackSM {
             }
           }
 
+          /**
+           * If either secondary has a Sync success rate for user below threshold, add it to `unhealthyReplicas` list
+           */
+          const userSecondarySyncMetrics = await SecondarySyncHealthTracker.computeUserSecondarySyncSuccessRates(
```

Comment:
can we move this logic into the peerSetManager file? It would be cleaner to have the peer health determination logic in this file as I have moved any peer health calculation in my reconfig pr into this file too

Reply:
i think this is actually fine where it is. the distinction is peerSetManager is abstracted on a node level whereas this is per user per node. if i understand correctly, the unhealthyReplicas is on a per user basis below.

Reply:
vicky i kinda see your point but this logic of secondarySyncHealth tracking only works from the primary's angle, and PeerSetMAnager doesn't really have a notion of that distinction right now

might make sense but given its like 8 lines, not sure if it makes sense to do right now